src/test/java/com/google/test/GoogleSeleniumTest.java

```diff
@@ -1,40 +1,31 @@
 package com.google.test;
 
-import io.github.bonigarcia.wdm.WebDriverManager;
+import com.google.test.util.Browser;
 import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeDriver;
 
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Target;
+import java.util.concurrent.TimeUnit;
 
 
 public class GoogleSeleniumTest {
 
-    @BeforeAll
-    public static void setupClass() {
-        WebDriverManager.chromedriver().setup();
-    }
+    public WebDriver driver = new Browser().browser();
```

Comment:
`Browser.browser()` is weird

The class should call `BrowserManager` the method should call `build` and take an input parameter with the browser name